content/en/dynamic_instrumentation/_index.md

```diff
@@ -37,17 +32,17 @@ Dynamic Instrumentation requires the following:
 
 - [Datadog Agent][1] 7.41.1 or higher is installed alongside your service.
 - [Remote Configuration][2] is enabled in that Agent.
-- For Java applications, tracing library [`dd-trace-java`][3] 1.8.0 or higher.
+- For Java applications, tracing library [`dd-trace-java`][3] 1.15.0 or higher.
```

Comment:
Let's also update minimum `Datadog Agent` version to `7.44.0` as discussed in slack.

Reply:
Done in 6a6e9133b97fffccfd5a4cebcb61b152394dd5e3